src/components/Typing/Typing.jsx

```diff
@@ -1,14 +1,19 @@
-import React, {useState, useEffect} from "react";
+import React, {useState, useEffect, useMemo} from "react";
 import style from "./Typing.module.css";
 import hljs from 'highlight.js';
 import ReactQuill from 'react-quill';
 import Button from "../Button/Button";
 import 'react-quill/dist/quill.snow.css';
 import 'highlight.js/styles/atom-one-light.css';
+import debounce from 'lodash.debounce';
```

Comment:
Can we import it `debounce from 'lodash/debounce'`

Reply:
The npm package name is `lodash.debounce`. How do I change that?